src/client/cl_main.cpp

```diff
@@ -773,7 +773,11 @@ static void CL_Record_f(void)
     {
         const char *s = Cmd_Argv(1);
         Q_strncpyz(demoName, s, sizeof(demoName));
```

Comment:
Should check if demo name is empty because if you click the Play button without selecting anything, that's what you get.

Reply:
@dGr8LookinSparky did you see this?

Reply:
@cengique , these particular lines are in the `CL_Record_f()` function, and there is a check right before those lines for a non-empty argument that would be used for the demo name, `if (Cmd_Argc() == 2)`.  The function that should be used for demo playback is `CL_PlayDemo_f()` and it has a check for a name argument: https://github.com/GrangerHub/tremulous/blob/f70166fc3c05381dedeb523e3282f84668f8ed55/src/client/cl_main.cpp#L1106 .  However, I just submitted a commit that prevents the demo playback menu from attempting to execute that command without a demo name selected, and opens a demo error popup menu instead:

https://github.com/GrangerHub/tremulous/commit/564bc9ee41fef1badec404ebe960d82e3c9acddf

Reply:
Tested, works.